### PR TITLE
Remove threading limitation to enable workers

### PR DESCRIPTION
**Pull request by jim68000**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-web: waitress-serve --listen=0.0.0.0:$PORT --threads=${WEB_CONCURRENCY:-4} server.wsgi:application
+web: waitress-serve --listen=0.0.0.0:$PORT server.wsgi:application
```
Comment (mcbhenwood) [inline Procfile:1]: Do you think that maybe Ravi intended that we just remove the `-4` to leave the threads == `WEB_CONCURRENCY`?

Comment (r4vi) [reply Procfile:1]: the `4` is a default if WEB_CONCURRENCY is not set - the `-` is a bash quirk, if you remove it I think waitress will use `4` anyway as that's the default, you'll just remove the ability to override it using WEB_CONCURRENCY env var

Comment (mcbhenwood) [reply Procfile:1]: skooled in bash, in public 🤦‍♂️

Comment (r4vi) [reply Procfile:1]: @mcbhenwood parameter substitution in bash always makes me want to :cry:  http://www.tldp.org/LDP/abs/html/parameter-substitution.html

Comment (jim68000) [reply Procfile:1]: Ah, then that won't do what I want anyway, as it's already running 4 threads. Closing.

Comment (mcbhenwood) [reply Procfile:1]: Sad thing is @r4vi I _used_ to know all that in my more platform-engineering focused days (and way before that in a previous life doing _A LOT_ of Linux admin). Which is why I'm facepalming now 😀

**Pull request closed without merging**